gravatar-quickeditor/src/main/java/com/gravatar/quickeditor/ui/editor/GravatarQuickEditorParams.kt

```diff
@@ -0,0 +1,66 @@
+package com.gravatar.quickeditor.ui.editor
+
+import java.util.Objects
+
+/**
+ * All non-auth params required to launch the Gravatar Quick Editor
+ *
+ * @property appName The appName of the app that is launching the Quick Editor
+ * @property email The email of the user
+ */
+public class GravatarQuickEditorParams private constructor(
+    public val appName: String,
+    public val email: String,
```

Comment:
We have a Gravatar `Email` class that we can use here. I have mixed feelings about whether we should or not. For consistency, I guess the answer is yes for consistency with other modules, but it is not a strong opinion.

Reply:
hmm we will need that email value (to display it) and this object doesn't expose it (only the hash) by design, right?

Reply:
oh it does, via the `toString()` function. I guess that works 👍